### PR TITLE
Fixing test failures on python 2.6/windows

### PR DESCRIPTION
**Pull request by wiredfool**
These two tests now succeed on Py2.6 on Windows.

### CODE REVIEW

File: Tests/test_imagewin_pointers.py [hunk 1]
```diff
@@ -72,7 +72,11 @@ def serialize_dib(bi, pixels):
         memcpy(bp, ctypes.byref(bf), ctypes.sizeof(bf))
         memcpy(bp + ctypes.sizeof(bf), ctypes.byref(bi), bi.biSize)
         memcpy(bp + bf.bfOffBits, pixels, bi.biSizeImage)
-        return bytearray(buf)
+        try:
+            return bytearray(buf)
+        except:
```
Comment (hugovk) [inline Tests/test_imagewin_pointers.py]: `except ValueError:`

File: Tests/test_image_getim.py [hunk 2]
```diff
@@ -10,7 +10,14 @@ def test_sanity(self):
         if py3:
             self.assertIn("PyCapsule", type_repr)
 
-        self.assertIsInstance(im.im.id, int)
+
+        try:
+            #py2.6
+            target_types = (int, long)
+        except:
```
Comment (hugovk) [inline Tests/test_image_getim.py]: `except AssertionError:`

File: Tests/test_image_getim.py [hunk 3]
```diff
@@ -10,7 +10,14 @@ def test_sanity(self):
         if py3:
             self.assertIn("PyCapsule", type_repr)
 
-        self.assertIsInstance(im.im.id, int)
+
+        try:
+            #py2.6
+            target_types = (int, long)
```
Comment (hugovk) [inline Tests/test_image_getim.py]: How would this fail on non-Py2.6? Isn't it just creating a tuple? Should the `try`/`except` be around the assert?

(Should also include the expected exception for the `except:`. (And a space after `#` in comments.))

Comment (wiredfool) [inline Tests/test_image_getim.py]: yeah, that was just going too quickly.

**Pull request merged**